conda-incubator/conda-store
# Respect worker log level config setting

## Description

This small PR updates the conda store worker to respect the `CondaStoreWorker.log_level` config parameter. Now users can see that the log level set for their works gets applied to the worker.

## Pull request checklist



## How to test

Before this change, the worker was always running at `logger.INFO` log level. For example, you can see in the log output
```
. . .
conda-store-worker-1  | [2024-10-19 01:41:55,960: INFO/MainProcess] Connected to redis://:**@redis:6379/0
conda-store-worker-1  | [2024-10-19 01:41:55,962: WARNING/MainProcess] /opt/conda/envs/conda-store-server/lib/python3.12/site-packages/celery/worker/consumer/consumer.py:508: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
```

To test this add the following line to your conda-store config file

```
c.CondaStoreWorker.log_level = logging.ERROR
```

Restart the server and view the logs. Notice that the WARN and INFO log lines no longer appear in the logs.
